### ISSUE TITLE
Release 4.0.3 on NPM seems to include the Git repo contents, not the built package!

### ISSUE THREAD

**Issue by tekacs** [microsoft/code-push#761]
It appears that `code-push@4.0.3` has been pushed to NPM as the contents of the Git repo, rather than the built output.

This is manifesting as it being impossible to require the package's contents, since "main" points to "script/index.js" which doesn't exist (there's no `script` dir!)

```
# If we install 4.0.2
~/tmp> npm install code-push@4.0.2

changed 1 package, and audited 100 packages in 370ms

8 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

# Then all the right files are there
~/tmp> l node_modules/code-push/
.rw-r--r-- 1.4k amar 26 Oct  1985 package.json
.rw-r--r--  596 amar 26 Oct  1985 plugin.xml
.rw-r--r-- 9.6k amar 26 Oct  1985 README.md
drwxr-xr-x    - amar  5 Apr  0:32 script
drwxr-xr-x    - amar  5 Apr  0:32 utils


# But if we install 4.0.3
amar@tekM1 ~/tmp> npm install code-push@4.0.3

changed 1 package, and audited 100 packages in 716ms

8 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

# ... then the contents of the Git repo are here, with no script/ folder.
amar@tekM1 ~/tmp> l node_modules/code-push/
drwxr-xr-x    - amar  5 Apr  0:32 bin
.rw-r--r-- 1.1k amar 26 Oct  1985 LICENSE.md
.rw-r--r-- 2.6k amar 26 Oct  1985 migration-notice.md
.rw-r--r--   38 amar 26 Oct  1985 mocha.opts
.rw-r--r-- 1.4k amar 26 Oct  1985 package.json
.rw-r--r--  596 amar 26 Oct  1985 plugin.xml
.rw-r--r-- 9.6k amar 26 Oct  1985 README.md
.rw-r--r-- 4.1k amar 26 Oct  1985 SDK.njsproj
drwxr-xr-x    - amar  5 Apr  0:32 src
.rw-r--r--  168 amar 26 Oct  1985 tsconfig-release.json
.rw-r--r--  374 amar 26 Oct  1985 tsconfig.json
```

Rolling back to 4.0.2 fixes the issue for me.

**Comment by smancebo** [comment]
same here

**Comment by andreidubov** [comment]
Hi guys,
This is a really big mistake from our side, now we have fixed it and released a new corrected version of code-push v4.0.4. In the future, we will try to be more careful with releases and will test them more closely. Microsoft will continue to support Code-push, there are no plans to deprecate it. Thanks to the community for the quick reaction to the problem, from our side we will check the repositories for critical errors and bugs more often and will fix all vulnerabilities in code-push repos soon.

Thank you for your understanding and we apologize for the inconvenience.
